common/source/docs/common-electro-permanent-magnet-V3.rst

```diff
@@ -0,0 +1,143 @@
+.. _common-electro-permanent-magnet-gripper:
```

Comment:
This is the same anchor as used by [our other emp topic](https://github.com/ArduPilot/ardupilot_wiki/blob/master/common/source/docs/common-electro-permanent-magnet-gripper.rst). Propose you rename the FILE to common-electro-permanent-magnet-nicodrone-emp-v3.rst and the anchor to  `_common-electro-permanent-magnet-gripper_nicodrone_epm_v3`